hawkular-inventory-parent/hawkular-inventory-api/src/test/java/org/hawkular/inventory/api/model/JsonTest.java

```diff
@@ -93,6 +94,7 @@ public void deserializeRawResource() throws Exception {
         assertThat(deserialized).isNotNull();
         assertThat(deserialized.getId()).isEqualTo("ID");
         assertThat(deserialized.getMetrics()).extracting(Metric::getDisplayName).containsExactly("metric");
+        assertThat(deserialized.getMetrics()).extracting(Metric::getExpression).containsExactly("sum($metric)");
```

Comment:
I don't think $metric is a thing, the expression should be resolved.

Reply:
yes, now it will be resolved.